actor/builtin/miner/miner_test.go

```diff
@@ -696,6 +699,116 @@ func TestMinerSubmitPoStPowerUpdates(t *testing.T) {
 	})
 }
 
+func TestMinerSubmitPoStVerification(t *testing.T) {
+	tf.UnitTest(t)
+
+	message := types.NewMessage(address.TestAddress, address.TestAddress2, 0, types.ZeroAttoFIL, "submitPoSt", nil)
+	var commR1, commR2, commR3 types.CommR
+	copy(commR1[:], th.MakeRandomBytes(32))
+	copy(commR2[:], th.MakeRandomBytes(32))
+	copy(commR3[:], th.MakeRandomBytes(32))
+
+	t.Run("Sends correct parameters to post verifier", func(t *testing.T) {
+		minerState := *NewState(address.TestAddress, address.TestAddress, peer.ID(""), types.OneKiBSectorSize)
+		minerState.ProvingPeriodEnd = types.NewBlockHeight(ProvingPeriodDuration(types.OneKiBSectorSize))
+		minerState.SectorCommitments = NewSectorSet()
+		minerState.SectorCommitments.Add(1, types.Commitments{CommR: commR1})
+		minerState.SectorCommitments.Add(2, types.Commitments{CommR: commR2})
+		minerState.SectorCommitments.Add(3, types.Commitments{CommR: commR3})
+
+		// The 3 sector is not in the proving set, so its CommR should not appear in the VerifyPoSt request
+		minerState.ProvingSet = types.NewIntSet(1, 2)
+		vmctx := th.NewFakeVMContext(message, minerState)
+
+		verifier := &th.FakeVerifier{Valid: true}
+		vmctx.TestVerifier = verifier
+
+		miner := Actor{Bootstrap: false}
+
+		testProof := []types.PoStProof{th.MakeRandomPoStProofForTest()}
+		_, err := miner.SubmitPoSt(vmctx, testProof, types.NewIntSet())
```

Comment:
nit: I've been using `types.EmptyIntSet()` to try to be more idiomatic.  In retrospect it is redundant to this call which is also pretty clear but it would be good to be consistent.  If you want to keep it this way and drop an issue for removing `EmptyIntSet` feel free.

Reply:
I'm unclear on the benefits of EmptyIntSet, but I've changed it to keep consistent.

Reply:
Yeah I think the benefits are minimal and it's reasonable to remove it in a small PR for what you originally had.  I added it for a clear name.